### PR TITLE
update sample dialects for 2023.1

### PR DESCRIPTION
**Pull request by xingyz**

### CODE REVIEW

File: examples/IBM Db2/dialect.yml [hunk 1]
```diff
@@ -329,6 +332,7 @@ map to field types:
 column types:
   integer: int
   float: float
+  fixed: decimal(38, 8)
```
Comment (chak7087) [inline examples/IBM Db2/dialect.yml:335]: shouldn't that be "decimal(31,8)" ?

Comment (xingyz) [reply examples/IBM Db2/dialect.yml]: good catch.  I just made the correction. Do you think we can merge this PR?

**Comment by chak7087** [comment]
Cool let me run quick rest tests  and finish my insights-qa PR

**Comment by xingyz** [comment]
@chak7087 
Let's merge this PR?

**Review by fran8806 — APPROVED** [review]

**Comment by chak7087** [comment]
@xingyz Please check Redshift Dialect failure:
```
{
    "error": {
        "subcode": 111406,
        "message": "sql dialect config file is invalid (while parsing a block mapping\n in 'reader', line 302, column 5:\n      - name: create timestamp from comp ... \n        ^\nexpected <block end>, but found Scalar\n in 'reader', line 309, column 7:\n          END\n          ^\n\n at [Source: (ZipFileInflaterInputStream); line: 309, column: 1] (through reference chain: com.esri.insights.connector.jdbc.dialect.SQLDialect[\"sql functions\"]->java.util.ArrayList[54]))",
        "code": 500,
        "params": {
            "error": "while parsing a block mapping\n in 'reader', line 302, column 5:\n      - name: create timestamp from comp ... \n        ^\nexpected <block end>, but found Scalar\n in 'reader', line 309, column 7:\n          END\n          ^\n\n at [Source: (ZipFileInflaterInputStream); line: 309, column: 1] (through reference chain: com.esri.insights.connector.jdbc.dialect.SQLDialect[\"sql functions\"]->java.util.ArrayList[54])"
        }
    }
}

```

**Comment by xingyz** [comment]
@chak7087 issue should be resolved. please try again

**Comment by chak7087** [comment]
@xingyz looks good merge

**Pull request merged**